java/bundles/org.eclipse.set.feature.table/src/org/eclipse/set/feature/table/internal/TableServiceImpl.java

```diff
@@ -129,6 +131,7 @@ public final class TableServiceImpl implements TableService {
 
 	private final Map<TableCompareType, TableDiffService> diffServiceMap = new ConcurrentHashMap<>();
 	private static final Queue<Pair<BasePart, Runnable>> transformTableThreads = new LinkedList<>();
+	private static final Set<TableInfo> cantTransformationTable = new HashSet<>();
```

Comment:
@TruongQuangSB Can we please rename that variable?

```suggestion
	private static final Set<TableInfo> nonTransformableTables = new HashSet<>();
```